doc/nrf/gs_programming.rst

```diff
@@ -86,6 +87,53 @@ You can download it from the following links:
    .. note::
    	In a multi-image build, this allows you to debug the source code of your application only.
 
+
+.. _gs_programming_manage:
+
+Managing projects with SES
+=========================
+
+Using SES it is possible to add/remove files to the application project using the SES.
+
+It is also possible to configure other settings for the application.
+
+In order to manage CMakeLists.txt files using SES - Nordic Edition, the tags
+
+    - ``# NORDIC SDK APP START``
+    - ``# NORDIC SDK APP END``
+
+must be surrounding the following CMake commands in the application CMakeLists.txt file.
+
+    - ``target_sources``
+    - ``target_compile_definitions``
+    - ``target_include_directories``
+    - ``target_compile_options``
+
+
+Adding files
+-----------------------
+
+To add files to a project, right click on the **Project 'app/libapp.a'** and select:
+- Add new file to CMakeLists.txt
+- Add existing file to CMakeLists.txt
+
+    .. figure:: images/ses_add_files.png
```

Comment:
Did you mean for this figure to be under the bullet point, or at top level? If top level, the leading whitespace should be removed.

In general I think the usual style in .rst is to put top level directives at column 0, so this applies elsewhere in the patch.

Reply:
correct, will remove indentation.